jupyter-on-gke/jupyter_config/config-selfauth.yaml

```diff
@@ -0,0 +1,104 @@
+#
+# For reference see the configuration reference and default values, but make
+# sure to refer to the Helm chart version of interest to you!
+#
+# Introduction to YAML:     https://www.youtube.com/watch?v=cdLNKUoMc6c
+# Chart config reference:   https://zero-to-jupyterhub.readthedocs.io/en/stable/resources/reference.html
+# Chart default values:     https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/HEAD/jupyterhub/values.yaml
+# Available chart versions: https://jupyterhub.github.io/helm-chart/
+hub:
+  image: 
+    name: <Repo name here>
+    tag: latest
+  config: 
+    JupyterHub:
+      authenticator_class: 'gcpiapjwtauthenticator.GCPIAPAuthenticator'
+    GCPIAPAuthenticator: 
+      header_name: 'X-Goog-IAP-JWT-Assertion'
+      expected_audience: '/projects/${project_number}/global/backendServices/${service_id}'
+  networkPolicy:
+    enabled: false
+
+prePuller:
+  hook:
+    enabled: false
+
+proxy:
+  chp:
+    networkPolicy:
+      enabled: false
+  service:
+    type: NodePort
+    annotations: 
+      beta.cloud.google.com/backend-config: '{"default": "config-default-test"}'
+  traefik:
+    networkPolicy:
+      enabled: false
+
+scheduling:
+  userScheduler:
+    enabled: false
+
+singleuser:
+  memory:
+    limit: 1G
+    guarantee: 1G
+  cpu:
+    limit: .5
+    guarantee: .5
+  nodeSelector:
+    iam.gke.io/gke-metadata-server-enabled: "true"
+  image:
+    name: jupyter/tensorflow-notebook
+    tag: python-3.10
+  startTimeout: 1000
+# More info on kubespawner overrides: https://jupyterhub-kubespawner.readthedocs.io/en/latest/spawner.html#kubespawner.KubeSpawner
+#   profile example:
+#   - display_name: "Learning Data Science"
+#      description: "Datascience Environment with Sample Notebooks"
+#      kubespawner_override:
+#           cpu_limit: .5
+#           cpu_guarantee: .5
+#           mem_limit: 1G
+#           mem_guarantee: 1G
+#        image: jupyter/datascience-notebook:2343e33dec46
+#        lifecycle_hooks:
+#          postStart:
+#            exec:
+#              command:
+#                - "sh"
+#                - "-c"
+#                - >
+#                  gitpuller https://github.com/data-8/materials-fa17 master materials-fa;
+  profileList:
+    - display_name: "Basic"
+      description: "Creates CPU VMs as the compute for notebook execution."
+      default: true
+    - display_name: "GPU T4"
+      description: "Creates GPU VMs (T4) as the compute for notebook execution"
+      kubespawner_override:
+        image: jupyter/tensorflow-notebook:python-3.10
+        extra_resource_limits:
+          # number of gpus needed on the node
+          nvidia.com/gpu: "2"
+      #  possible values: nvidia-tesla-k80, nvidia-tesla-p100, nvidia-tesla-p4, nvidia-tesla-v100, nvidia-tesla-t4, nvidia-tesla-a100, nvidia-a100-80gb, nvidia-l4
+        node_selector:
+          cloud.google.com/gke-accelerator: "nvidia-tesla-t4"
+    - display_name: "GPU T4"
```

Comment:
What's the difference between these? Should one be A100 instead?

Reply:
Fixed. We should also make sure that the platform will crate a100 gpu nodes as well for standard right?